.github/workflows/continuous-integration.yaml

```
@@ -25,7 +25,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
```

We could use LTS to avoid save us the future updates

```suggestion
          node-version: lts/*
```